sphinx/frontend-bundling.rst

```diff
@@ -0,0 +1,20 @@
+Frontend Bundling
+=================
```

Comment:
```suggestion
***************************
Frontend Bundling (Webpack)
***************************
```
I suggest to keep the heading format consistent to the other .rst files and include the name of the main technology after the concept name.